style/preview.less

```diff
@@ -83,13 +83,13 @@
 
 	&-body {
 		max-height: 248px;
-		overflow: hidden;
+		overflow: scroll;
 
 		p {
 			margin: 0;
-			line-height: 1.6;
+			line-height: 1.7;
 			font-size: 18px;
-			padding: 10px 20px;
+			padding: 5px 20px;
```

Comment:
from the design sharing in the phab i can see the padding should remain as padding: 10px 20px; can you confirm that again?

Reply:
Updated 👍